build.sbt

```diff
@@ -5,7 +5,7 @@ organization := "io.shiftleft"
 ThisBuild / scalaVersion := "2.13.5"
 // don't upgrade to 2.13.6 until https://github.com/com-lihaoyi/Ammonite/issues/1182 is resolved
 ThisBuild /Test /fork := true
-val cpgVersion = "1.3.317"
+val cpgVersion = "1.3.326+0-09c1efd8+20210915-1021"
```

Comment:
```suggestion
val cpgVersion = "1.3.326"
```

Reply:
i didn't mention this, but this depends on https://github.com/ShiftLeftSecurity/codepropertygraph/pull/1405, so will update this with the "next version" once released.